docs/UserGuide.md

```diff
@@ -100,8 +103,9 @@ Adds a contact into Teaching Assistant.
 
 Format: `cadd n/NAME p/NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
 
-* A contact can have any number of tags (including 0).
-* A contact can be tagged to an entry to easily filter for contacts related an entry.
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A contact can be tagged to an entry to easily filter for contacts related an entry.
```

Comment:
Think this phrasing might be a bit misleading, should we remove it?

Reply:
If i say "an entry's name can be tagged to a contact to easily filter for contacts related an entry" is it clearer? cuz this in response to a pe bug where ppl say they cannot link contacts and entries

Reply:
Sure, looks good thanks!